components/DogCard/index.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+const DogCard = () => {
+  return (
+    <div>
+      <h1>Anton</h1>
```

Comment:
This is not a syntax for an array. But you should use an array to be able to create your content dynamically. Also it's not the right syntax to create a component like we learned it in our sessions. 

Your `DogCard` component should be written more like this, or? The component accepts the prop `dog`, which you pass through from the `_app.js` to the `DogCard` component:  

```
export default function DogCard({ dog }) {
  return (
    <li>
      <h2>{dog.name}</h2>
      <p> {dog.dateOfBirth}</p>
      <p> {dog.placeOfBirth}</p>
      <p>{dog.sex}</p>
      <p>{dog.color}</p>
      <p>{dog.breed}</p>
    </li>
  );
}

```